src/tools/starlark/java/com/google/devtools/starlark/common/DocstringUtils.java

```diff
@@ -479,15 +479,21 @@ private String checkForNonStandardDeprecation(String line) {
     /** Parses additional lines that can come after "param: foo" in an 'Args' section. */
     private void parseContinuedParamDescription(
         int baselineIndentation, StringBuilder description) {
+      // continuations are de-dented to the minimum value seen
+      int continuationIndentation = Integer.MAX_VALUE;
       while (nextLine()) {
         if (line.isEmpty()) {
           description.append('\n');
           continue;
         }
-        if (getIndentation(line) <= baselineIndentation) {
+        int lineIndentation = getIndentation(line);
+        if (lineIndentation <= baselineIndentation) {
           break;
         }
-        String trimmedLine = line.substring(baselineIndentation);
+        if (lineIndentation < continuationIndentation) {
```

Comment:
We're not truly finding the minimum indentation value here, as we're still processing as we go.
Consider the difference in behavior for:

```
  Line one
          Line two
                    Line three
```

versus

```
               Line one
        Line two
   Line three
```

Reply:
correct, but the failure mode is that you get some extra whitespace if the continuation is accidentally not consistent. For your example I think

```
Line one
        Line two
                  Line three
```
and
```
Line one
Line two
Line three
```
are the best outcome we'll get without scanning ahead for the minimal intent. Do you think it's worth adding the complexity to scan ahead?

Reply:
Yes, I don't think it's much more complex to scan ahead (we're not dealing with gigantic strings here, so it should be fine to store all the lines in a list, take the min indent, and then iterate over)

I foresee later user confusion / frustration with the current behavior, if we don't fix this right the first time.

Thanks!

Reply:
Okay, done PTAL